pages/common/set.md

```diff
@@ -0,0 +1,19 @@
+# set
+
+> Display, Set or Unset value of shell attributes and positional parameters.
```

Comment:
Not sure that `Set` and `Unset` need to be capitalised here.

Reply:
yes sure...